### DISCUSSION TITLE
Meeting in ical format

### DISCUSSION THREAD

**Discussion by dougie-wougie** [code4recovery/12-step-meeting-list#1420] [Ideas]
At our Central Office, I would like the meetings to be in .ical or a similar format to create a Google calendar of all the meetings. When the phone rings and we give out the next meeting, it would be nice to look at a calendar. It would be a nice feature. 

D

**Comment by seanalansnyder** [comment]
Hi Dougie,

Thank you for suggesting this new feature.
We will need to discuss the feasibility of implementing this.

In the interim, you may want to check our the [tsml_next_meetings] shortcode functionality that we currently have:
https://wordpress.org/plugins/12-step-meeting-list/#are%20there%20shortcodes%3F

Many groups have found it quite helpful place this shortcode somewhere on their homepage to serve as a quick reference like you describe.


Sean

**Comment by dougie-wougie** [comment]
Thank you. I use that as an interim. We use the printed meeting guide mostly.

**Comment by seanalansnyder** [comment]
Following up on this thread, we have discussed this feature.   

We are currently trying to gauge demand for this feature.  If there are any others who are looking for similar functionality, please add a "Me Too!" on this discussion thread!     

If there is sufficient demand, we would likely request a quick Zoom, @dougie-wougie, to make sure we fully understand the requirements/request.

**Comment by 12steprecovery** [comment]
Not sure if this is still an active discussion. 

This would be very helpful to transition from Events Manager plugin which we use on slaauk.org - we offer iCal and Google Calendar feeds of our meeting lists eg for the list of online video meetings https://slaauk.org/events/categories/zoom-12-step-meeting/ical which fellows can add to their own Apple Calendar. Also for the list of Intergroup meetings https://slaauk.org/events/categories/intergroup/ical

**Reply by joshreisner** [reply]
hi @12steprecovery 

we have a feature with our TSML UI appearance where members can add individual meetings to their calendars ([example](https://aasanjose.org/meetings/a-vision-for-you-2))

<img width="704" height="702" alt="Screenshot 2026-04-04 at 8 10 56 AM" src="https://github.com/user-attachments/assets/ff739cec-95d4-4e83-a186-2bea437ae2e8" />

we don't yet have a way for members to subscribe to ALL the meetings on the website in a single calendar feed - as per the original request - we think this might be more of an edge-case

also, we recommend sites use a different mechanism for listing service / business meetings - TSML is really just for recovery. one option that many WP sites use is https://theeventscalendar.com

**Reply by 12steprecovery** [reply]
I appreciate that it's possible to add individual meetings to a calendar. As I said we are currently using Events Manager Plugin https://wp-events-plugin.com/ which allows to create all sorts of feeds, including, for example a feed for all Zoom meetings which I can add to my phone calendar. This is not really an edge case I would argue as it provides a useful resource for fellows. 

There's also a teamup calendar created by SLAA Virtual Intergroup https://teamup.com/ksxi176ztzzuswm5pk that has the option to add ical feeds for any of the types of meetings.  it would be great to reproduce this facility, which could also be useful for districts - so that any new face to face meeting just appears in the list!
